models/stg_hubspot__ticket.yml

```diff
@@ -49,6 +49,8 @@ models:
         description: '{{ doc("_fivetran_synced") }}'
       - name: _fivetran_start
         description: The (UTC DATETIME) to keep track of the time when a record is first created or modified in the source database.
+      - name: _fivetran_end
```

Comment:
We should probably add this to the `src_hubspot.yml` as well. It looks as if `_fivetran_start` is not in the `src_hubspot.yml` either.